components/toast/toast.test.tsx

```diff
@@ -20,39 +20,41 @@ describe('components/Toast', () => {
     };
 
     test('should match snapshot for showing toast', () => {
-        const wrapper = shallow(<Toast {...defaultProps}><span>{'child'}</span></Toast>);
+        const wrapper: ShallowWrapper<any, any, Toast> = shallow(<Toast {...defaultProps}><span>{'child'}</span></Toast>);
```

Comment:
For these regular `shallow` calls, you can save a bit of typing by telling TS the type of `shallow` instead of the type of `wrapper`.

```suggestion
        const wrapper = shallow<Toast>(<Toast {...defaultProps}><span>{'child'}</span></Toast>);
```

I don't think that works for `mountWithIntl` though so you don't need to change that one